plutus-conformance/src/PlutusConformance/Common.hs

```diff
@@ -191,75 +212,95 @@ getTestedValue eval file = do
+          <> "\n But the expected result, with the unique names shown is: \n"
+          <> show expected
 compareAlphaEq (Right expected) (Left actualTxt) =
-    pure $ "Test failed, the output failed to parse or evaluate: \n"
-        <> T.unpack actualTxt
-        <> "\n But the expected result, with the unique names shown is: \n"
-        <> show expected
+  pure $
+    "Test failed, the output failed to parse or evaluate: \n"
+      <> T.unpack actualTxt
+      <> "\n But the expected result, with the unique names shown is: \n"
+      <> show expected
 compareAlphaEq (Left txt) (Right actual) =
-    {- this is to catch the case when the expected program failed to parse because
-    our parser doesn't support `data` atm. In this case, if the textual program is the same
-    as the actual, the test succeeds. -}
-    if txt == display actual then Nothing
-    else Just $
-        "Test failed, the output was successfully parsed and evaluated, but it isn't as expected. "
-        <> "The output program is: "
-        <> display actual
-        <> ". But the expected result is: "
-        <> T.unpack txt
+  {- this is to catch the case when the expected program failed to parse because
+  our parser doesn't support `data` atm. In this case, if the textual program is
+  the same as the actual, the test succeeds. -}
+  if txt == display actual
+    then Nothing
+    else
+      Just $
+        "Test failed, the output was successfully parsed and evaluated, \
+        \but it isn't as expected. "
+          <> "The output program is: "
+          <> display actual
+          <> ". But the expected result is: "
+          <> T.unpack txt
 
--- | Update the golden file with the tested value. TODO abstract out for other tests.
-updateGoldenFile ::
-    FilePath -- ^ the path to write the golden file to
-    -> Either T.Text UplcProg -> IO ()
+{-| Update the golden file with the tested value.
+TODO abstract out for other tests.
+-}
+updateGoldenFile
+  :: FilePath
+  -- ^ the path to write the golden file to
+  -> Either T.Text UplcProg
+  -> IO ()
 updateGoldenFile goldenPath (Left txt) = T.writeFile goldenPath txt
 updateGoldenFile goldenPath (Right p)  = T.writeFile goldenPath (display p)
 
--- | Run the UPLC evaluation tests given an `evaluator` that evaluates UPLC programs.
-runUplcEvalTests ::
-    UplcEvaluator -- ^ The action to run the input through for the tests.
-    -> (FilePath -> Bool)
-    -- ^ A function that takes a test name and returns
-    -- whether it should labelled as `ExpectedFailure`.
-    -> (FilePath -> Bool)
-    -- ^ A function that takes a test name and returns
-    -- whether it should labelled as `ExpectedBudgetFailure`.
-    -> IO ()
+{-| Run the UPLC evaluation tests given an `evaluator` that evaluates UPLC
+programs.
+-}
+runUplcEvalTests
+  :: UplcEvaluator
+  -- ^ The action to run the input through for the tests.
+  -> (FilePath -> Bool)
+  {- ^ A function that takes a test name and returns
```

Comment:
Why `{- ^` with the space, but `{-|` without the space? I'd prefer the latter to have a space too.

Reply:
This is because in our fourmolu config we have `haddock-style: multiline-compact`, could be fixed by `haddock-style: multiline`.